regsync/regsync/app.py

```diff
@@ -14,30 +15,28 @@ def main():
     level = logging.getLevelName(LOG_LEVEL)
     logging.basicConfig(level=level)
 
-    REGISTRY_URI = os.environ.get("APP_REGISTRY_URI")
-    if REGISTRY_URI is None:
-        logger.error("No APP_REGISTRY_URI specified.")
-        sys.exit(1)
+    MODEL_CACHE_PATH = os.environ.get("REGSYNC_MODEL_CACHE_PATH", "./models")
```

Comment:
I am a little concerned we could end up nuking an import folder `models` that we don't own...

I think the default should be `brdg-models` to avoid this